backend/python/tapa/verilog/xilinx/module.py

```diff
@@ -344,6 +348,14 @@ def func(item: ast.Node) -> bool:
 
     self._filter(func, 'param')
 
+  def add_instancelist(self, item: ast.InstanceList) -> 'Module':
```

Comment:
Why do you create `ast.InstanceList` explicitly and use this variant, instead of using the existing `add_instance` method? I understand the latter always inserts a pragma, but that can be made configurable by adding a new argument.

Reply:
I start with an existing InstanceList from the ast